src/language/walker/ruleWalker.ts

```diff
@@ -68,21 +64,14 @@ export class RuleWalker extends SyntaxWalker {
         }
     }
 
-    public skip(node: ts.Node) {
```

Comment:
I'm really hope nobody is using this. It looks like it implemented with a bunch of other stuff and never used